docs/synapse/devopsguides/ingest_intro.rst

```diff
@@ -0,0 +1,641 @@
+Ingest Subsystem
```

Comment:
I think it might make more sense to put it under user docs, since at least so far, the docs have seemed to focus on how analysts/users can use synapse.lib.ingest to load data.